open_seq2seq/parts/cnns/conv_blocks.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2018 NVIDIA Corporation
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import tensorflow as tf
+
+def conv_bn_actv(layer, name, inputs, filters, kernel_size, activation_fn, strides,
+                   padding, regularizer, training, data_format, bn_momentum,
+                   bn_epsilon):
+  """Helper function that applies convolution, batch norm and activation."""
+  conv = layer(
+    name="{}".format(name),
+    inputs=inputs,
+    filters=filters,
+    kernel_size=kernel_size,
+    strides=strides,
+    padding=padding,
+    kernel_regularizer=regularizer,
+    use_bias=False,
+    data_format=data_format,
+    )
```

Comment:
Another codestyle suggestion: the closing bracket should be either aligned with the first statement (conv = ...) or be on the previous line: data_format=data_format). Same for the bn block